x-pack/plugin/eql/src/test/resources/querytranslator_tests.txt

```diff
@@ -93,13 +93,14 @@ mixedTypeFilter
 process where process_name : "notepad.exe" or (serial_event_id < 4.5 and serial_event_id >= 3.1)
 ;
 "term":{"process_name":{"value":"notepad.exe","case_insensitive":true,"boost":1.0}
-"range":{"serial_event_id":{"from":3.1,"to":4.5,"include_lower":true,"include_upper":false
+{"bool":{"must":[{"range":{"serial_event_id":{"from":null,"to":4.5,"include_lower":false,"include_upper":false
+{"range":{"serial_event_id":{"from":3.1,"to":null,"include_lower":true,"include_upper":false
 ;
 
 notFilter
 process where not (exit_code > -1)
 ;
-"range":{"exit_code":{"from":null,"to":-1,"include_lower":false,"include_upper":true
+{"bool":{"must_not":[{"range":{"exit_code":{"from":-1,"to":null,"include_lower":false,"include_upper":false
```

Comment:
👍  was just looking around for this. a conjunction like `not (exit_code > 1 and user_name == "root")` is equivalent to `not (exit_code > 1) and not (user_name == "root")` then?

Reply:
No, they are not equivalent and the queries that get created reflect this difference. Subtle, but it's there.

Let's assume `exit_code = 100` and `user_name = "x"`
- `not(100 > 1 and "x" == "root")` => `not(true and false)` => `true`
- `not(100 > 1) and not("x" == "root")` => `not true and not false` => `false and true` => `false`

Reply:
oh, sorry. I failed my DeMorgan's logic. I think I meant `not (exit_code > 1) or not (user_name == "root")`

Reply:
I think that, yes, they should be equivalent. But we don't do anything to simplify the logic. For the same reason as https://github.com/elastic/elasticsearch/pull/74567#discussion_r660492515, meaning `not` should be there as we act on its presence accordingly.